tests/AulaTests.hs

```diff
@@ -33,11 +33,21 @@ import Servant          as X
 import Frontend         as X
 import Frontend.Testing as X
 import Frontend.Prelude as X hiding (get, put)
+import Test.Hspec.Missing as X
+
 import Arbitrary (constantSampleTimestamp)
 import Logger (LogLevel(..), nullLog)
 import Persistent (mkMetaInfo)
 
 
+-- Be default, test cases are part of the smoke test suite.
+data TestSuite
+    = Large
+  deriving (Eq, Show)
+
+instance Tag TestSuite where
+    tagText = cs . show
```

Comment:
default instance!  (-:

it requires default signature again though, so @andorp you are probably against it, and i don't insist.  just saying it'd be fun...